sdk/commands/sensor_test.ts

```diff
@@ -1,27 +1,38 @@
-import { StructBuffer, bitFields, bits, bool, int16_t, uint16_t, uint8_t } from "@nmann/struct-buffer";
-import { API_COMMAND, char2byte } from "../api";
+import { StructBuffer, bitFields, bits, int16_t, uint16_t, uint8_t } from "@nmann/struct-buffer";
+import { API_COMMAND, PANEL_COUNT, SENSOR_COUNT } from "../api";
 import type { Decoded } from "./config";
-import type { EachPanel, EachSensor } from "./inputs";
 
 /**
  * Sensor Test Mode values the stages expect
  */
-export const SENSOR_TEST_MODE = {
+export enum SensorTestMode {
```

Comment:
Converted this to an Enum. Not sure it matters too much, I think they just end up just being `consts` anyway right? I like the idea of assigning an argument as an Enum type instead of just saying `number`

Reply:
yeah... they kind of become consts, but not really. they're honestly a weird relic of the early days of typescript and there are some foot-gun type properties about them. this is fine for our purposes though.